File: 5-Matematica/1214-Acima-da-Media/1214.c
#include <stdio.h>

int main() {

    int n, alu;
    scanf ("%d", &n);
    for (int i = 1; i <= n; i++) {
        int notas[1001] = {};
        scanf ("%d", &alu);
        for (int q = 1; q <= alu; q++) {
            scanf ("%d", &notas[q]);
            notas[0] += notas[q];
        }
        int cont = 0;
        for (int q = 1; q <= alu; q++) {
            if (notas[q] > (float) (notas[0] / alu)) {
                cont ++;
            }
        }
        printf ("%.3f%\n", ((float)cont / alu) * 100);
    }

    return 0;
}

File: 5-Matematica/1214-Acima-da-Media/1214.py
for _ in range (int(input())):
    notas = input().split()
    notas = [int(i) for i in notas]
    notas.pop(0)
    media = sum(notas)/len(notas)
    cont = 0
    for i in notas:
        if i > media:
            cont += 1
    cont = cont/len(notas) * 100
    cont = ("%.3f" % cont)
    print ("{}{}". format(cont,"%"))